pkgs/by-name/an/antora-ui-default/package.nix

```diff
@@ -0,0 +1,71 @@
+{
+  fetchFromGitLab,
+  lib,
+  stdenvNoCC,
+}:
+let
+  srcFetchFromGitLab = {
+    hash = "sha256-q2FwkwzjanxTIxjMpCyMpzPt782uYZiWVdZ7Eev79oM=";
+    owner = "trueNAHO";
+    repo = "antora-ui-default";
+    rev = "83bf9bf5f22a6dee397f8b089eb0315c14a278b5";
+  };
```

Comment:
No need to pull this into a local. Just use it down below.

Reply:
Yes, this use of a let binding feels a little superfluous. Setting it directly inside the fetchFromGitLab call attributes would be a bit clearer.

Reply:
According to https://github.com/NixOS/nixpkgs/pull/332341#discussion_r1790617183, I will inline the `pname` value. However, the `srcFetchFromGitLab` values are also used to generate the following documentation URL: 

https://github.com/NixOS/nixpkgs/blob/b63ef6df743c3164c259524875c896273d0008d5/pkgs/by-name/an/antora-ui-default/package.nix#L34

Should I inline all `srcFetchFromGitLab` instances or only `pnanme`?